pylint_odoo/checkers/modules_odoo.py

```diff
@@ -237,6 +237,20 @@ class ModuleChecker(misc.WrapperModuleChecker):
                      'javascript lint. You can use the environment variable '
                      '"PYLINT_ODOO_JSLINTRC" too. Default: %s' % DFTL_JSLINTRC)
         }),
+        ('po-ignore', {
```

Comment:
I try to use `--ignore` but enter in conflict with `ignore` of pylint
It is a good solucion?